src/main/java/javaserver/ResponseBuilders/DirectoryResponseBuilder.java

```diff
@@ -0,0 +1,42 @@
+package javaserver.ResponseBuilders;
+
+import javaserver.RequestHandlers.DirectoryHandler;
+import javaserver.HTMLContent;
+import javaserver.HTTPStatusCode;
+import javaserver.Routes;
+
+public class DirectoryResponseBuilder extends ResponseBuilder {
+	DirectoryHandler requestHandler;
```

Comment:
I think it's a good idea to get into the habit of defining access modifiers for properties to be consistent. It seems like you omit them sometimes, and add them other times.